### DISCUSSION TITLE
How to run "react-email-client" in a local dev server?

### DISCUSSION THREAD

**Discussion by blint46** [resend/react-email#1103] [Q&A]
Hi all,

I'm trying to run the `"react-email-client" `package locally to but it is not defined in the `pnpm-workspace.yaml` as a workspace nor any lock file inside the "client" directory.

How should I run the package locally?

**Comment by taylor-lindores-reeves** [comment]
@c-h-o-n please check [my ticket](https://github.com/resend/react-email/discussions/1106) and thumbs up if you agree.
